packages/switch/src/spectrum-switch.css

```diff
@@ -140,7 +211,7 @@ governing permissions and limitations under the License.
 }
 
 #input:focus-visible + #switch:after {
-    box-shadow: 0 0 0 var(--mod-switch-focus-indicator-thickness, var(--mod-focus-indicator-thickness, var(--spectrum-switch-focus-indicator-thickness))) var(--highcontrast-switch-focus-indicator-color, var(--mod-switch-focus-indicator-color, var(--spectrum-switch-focus-indicator-color)));
```

Comment:
I just want to note here that --mod-focus-indicator-thickness is not removed, it's referenced on line 25 which results in the same modifier rendering order as the previous stack supported here.